com.gooseberrydevelopment.pinepods.yml

```diff
@@ -0,0 +1,44 @@
+id: com.gooseberrydevelopment.pinepods
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+command: app
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - ../../shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/madeofpendletonwool/PinePods/releases/download/0.7.1/Pinepods_0.7.1_amd64.deb
+        sha256: 0638853425d8831ec67aef59ed84ae55b55e409110fda269f9fdb90dd817e83c
+        only-arches: [x86_64]
+        dest-filename: pinepods.deb
+      - type: file
+        path: icons/32x32.png
+      - type: file
+        path: icons/128x128.png
+      - type: file
+        path: icons/256x256.png
+      - type: file
+        path: ./com.gooseberrydevelopment.pinepods.metainfo.xml
+
+    build-commands:
+      - ar -x pinepods.deb
+      - tar -xf data.tar.gz
+      - sed -i 's/Icon=app/Icon=com.gooseberrydevelopment.pinepods/' usr/share/applications/Pinepods.desktop
+      - sed -i 's/Exec=app/Exec=app/' usr/share/applications/Pinepods.desktop
```

Comment:
don't use sed. use the command `desktop-file-edit`.

Reply:
Switch to desktop-file-edit